tools/coredumper/sof-coredump-reader.py

```diff
@@ -56,7 +56,7 @@ def stdout_print(*args, **kwargs):
 	]]
 ]
 
-# Exception casues:
+# Exception causes:
```

Comment:
I can see why you are doing this, but this is bumping up the review effort. I'm mentally scanning for changes that only cover platforms, so this kind of stuff jumps out.

Reply:
+1

Reply:
it is related to this patch - it removes "sue" ;-) Obviously, that's why I hit this and other typos fixed in this PR, personally I'd keep these changes - they are tiny and obvious comments, but can trivially be extracted too...